src/loadtest/hibp.js

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * NOTE: This is a k6 loadtest script, see https://k6.io/ for more information.
+ *
+ * In particular, k6 supports JS but is not a Node package: https://k6.io/docs/get-started/installation/
+ * This is why import/no-unresolved and no-undef are ignored.
+ */
+
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-undef */
+
+import http from "k6/http";
```

Comment:
Is there a doc for how to use this script?

Reply:
It's pretty configurable at runtime, maybe best place to link might be https://grafana.com/docs/k6/latest/get-started/running-k6/#add-vus

Reply:
Or just https://grafana.com/docs/k6/latest/get-started/running-k6

Reply:
Do you often use different configurations, or is there a specific set of parameters you usually call it with? If the latter, would be useful to have an example in the docblock above, or maybe even an addition to `scripts` in `package.json`?

Reply:
Adding this to `scripts` in `package.json` is a great idea - the main part that you generally want to configure at runtime is the number of virtual (concurrent) users.

Reply:
One problem with calling from `package.json` is that `k6` is a native (Go language) binary that needs to be installed separately, so it could be a little confusing. k6 uses Go to load this `hibp.js` script, which makes it unlike our other JS code that expects to run under browser and/or Node.

Reply:
Yeah fair enough, as long as there's an example command somewhere we can use as an example. I'd normally also ask for the longform version of CLI arguments, but given that the longform of `-u` is apparently `--vus`, that's not necessarily more informative :sweat_smile: